sonarqube/resource_sonarqube_project_test.go

```diff
@@ -304,19 +304,19 @@ func TestAccSonarqubeProjectSettingsUpdate(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(name, "project", "testAccSonarqubeProject"),
 					resource.TestCheckResourceAttr(name, "setting.#", "1"),
-					resource.TestCheckResourceAttr(name, "setting.0.key", "sonar.demo"),
-					resource.TestCheckResourceAttr(name, "setting.0.value", "sonarqube@example.org"),
+					resource.TestCheckResourceAttr(name, "setting.0.key", "sonar.docker.activate"),
+					resource.TestCheckResourceAttr(name, "setting.0.value", "false"),
 				),
 			},
 		},
 	})
 }
 
-func TestAccSonarqubePortfolioSettingsTypes(t *testing.T) {
+func TestAccSonarqubeProjectSettingsTypes(t *testing.T) {
```

Comment:
Does this mean there are no longer portfolio-level tests or was the test mis-named?

Reply:
This PR is a fix of a previous pr that introduced this functionality.
This name was not correct, I fixed now.